lib/src/config_util.cc

```diff
@@ -0,0 +1,29 @@
+#include <internal/config_util.hpp>
+#include <cctype>
+
+namespace hocon {
+
+    bool is_whitespace(char c) {
+        switch (c) {
+            case ' ':
+            case '\n':
+            // TODO: these complain about char overflow, what do.
```

Comment:
Places I suggest starting to read up on are [ICU4C](http://site.icu-project.org/). Also, Java uses UTF-16; we need to watch for assumptions the Java code might make related to that.